packages/trader/src/exchange-adapters/UniswapV3Trader.ts

```diff
@@ -0,0 +1,59 @@
+import winston from "winston";
+import Web3 from "web3";
+import BigNumber from "bignumber.js";
+
+const bn = require("bignumber.js"); // Big number that comes with web3 does not support square root.
+bn.config({ EXPONENTIAL_AT: 999999, DECIMAL_PLACES: 40 });
+
+const { encodePriceSqrt } = require("@uma/common");
+const ExchangeAdapterInterface = require("./ExchangeAdapterInterface");
+const { getTruffleContract } = require("@uma/core");
```

Comment:
Wherever we can, we should ideally be using `import X from Y;` syntax. If this isn't possible with core and common, it should at least be allowed for ExchangeAdapterInterface since that's a ts file.

Reply:
yup! will update